public/app/features/admin/UserOrgs.tsx

```diff
@@ -224,9 +225,21 @@ export class AddToOrgModal extends PureComponent<AddToOrgModalProps, AddToOrgMod
     const { isOpen } = this.props;
     const { role } = this.state;
     const styles = getAddToOrgModalStyles();
-
+    const modalStyles = getModalStyles(this.props.theme);
+    const contentClassName = cx(
+      modalStyles.modalContent,
+      css`
+        overflow: visible;
+      `
+    );
```

Comment:
```suggestion
     const contentClassName = css`
      overflow: visible;
    `;
```

The modal content already has content styles applied, so this shouldn't be necessary here.

Reply:
Yes, I realized that when I saw Torkel's comment. Will fix ASAP.